src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -648,6 +648,12 @@ namespace AppInstaller::CLI::Workflow
         ReportIdentity(context, manifest.CurrentLocalization.Get<Manifest::Localization::PackageName>(), manifest.Id);
     }
 
+    void LogManifestFields(Execution::Context& context)
+    {
+        const auto& manifest = context.Get<Execution::Data::Manifest>();
+        Logging::Telemetry().LogManifestFields(manifest.Id, manifest.DefaultLocalization.Get<Manifest::Localization::PackageName>(), manifest.Version);

```

Comment:
>LogManifestFields [](http://example.com/codeflow?start=29&length=17)

nit: usually this gets logged when manifest is generated(you can find them in all other places). I think we could remove this task and just do the same in PackageManager.cpp as well, or at least update all other places to call this workflow task.